lib/utils/helpers.ts

```diff
@@ -25,6 +25,9 @@ import * as ShellEscape from 'shell-escape';
 import { ExpectedError } from '../errors';
 import { getBalenaSdk, getChalk, getVisuals } from './lazy';
 
+import * as semver from 'balena-semver';
```

Comment:
The plain `semver` package (as opposed to `balena-semver`) is sufficient for `semver.gte`, right? I worry that this could slow down CLI start up time by loading unnecessary packages. See:
* https://www.flowdock.com/app/rulemotion/i-cli/threads/kW6PdVia9J41YOM6R7WBKLzQMl8
* https://github.com/balena-io/balena-cli/pull/1839

Reply:
(`balena-semver` requires `semver`)

Reply:
Ok changed